js/main.js

```diff
@@ -45,7 +33,50 @@ if (shouldQuit) {
 app.on('ready', getUrlAndOpenMainWindow);
 
 function getUrlAndOpenMainWindow() {
-    // for dev env allow passing url argument
+    let installMode = false;
+    
+    process.argv.some((val) => {
+        let flag = '--install';
+        if (val === flag) {
+            installMode = true;
+            getConfigField('launchOnStartup')
+            .then(setStartup);
+        }
+
+        return false;
+    });
+
+    if (installMode === false){
+        openMainWindow();
+    }
+}
+
+function setStartup(lStartup){
+    if (lStartup === true){
+        symphonyAutoLauncher.isEnabled()
```

Comment:
By simplifying this, registry settings do not function correctly on .msi call. I´ve tried several approaches to make it simpler but this is the only form that Registry was done on .msi call and this is how demo code instructs to use auto-launch library.